### PR TITLE
[CNVS Upgrade] [React router migration] Migrate named routes

### PR DESCRIPTION
**Pull request by nLight**
This PR is the first part of the react-router migration. It removed all named routes and replaces them with absolute paths. There could still be bugs here and there even though I clicked through the whole thing. Unfortunately I couldn't find a way to deliver this part in smaller parts.

https://mesosphere.atlassian.net/browse/DCOS-10646

⚠️  Remember the plugins ☝️

### CODE REVIEW

**Comment by MatApple** [comment]
retest this please

**Comment by nLight** [comment]
@mesosphere-ci retest this please

File: plugins/services/src/js/components/TaskTable.js [hunk 1]
```diff
@@ -287,11 +287,11 @@ class TaskTable extends React.Component {
...
     let routeParams = Object.assign({taskID: task.id}, params);
 
-    let linkTo = 'services-task-details-files-viewer';
+    let linkTo = '/services/overview/:id/tasks/:taskID/view';
     if (params.nodeID != null) {
-      linkTo = 'nodes-task-details-files-viewer';
+      linkTo = '/nodes/:nodeID/tasks/:taskID/view';
     }
-
+    console.log(linkTo);
```
Comment (jfurrow) [inline plugins/services/src/js/components/TaskTable.js]: `console.log` :)

**Comment by jfurrow** [comment]
This works well, nice job!

**Comment by philipnrmn** [comment]
I can no longer route to node health checks in the table, they all link to the current page. They previously linked to but this path no longer works. 
`nodes/:nodeID/health/:nodeIPAddress/:serviceName/`

File: plugins/services/src/js/components/TaskTable.js (hunk 1 above)
Comment (nLight) [inline plugins/services/src/js/components/TaskTable.js]: hehe, 🔥 that one, thanks!

**Comment by nLight** [comment]
Fixing rebase artifacts thus `hold merge`

**Pull request merged**